data/Language/JavaScript.yaml

```diff
@@ -1,6 +1,7 @@
 identifier: JavaScript
 wikidata: Q2005
-grammar_file: https://github.com/antlr/grammars-v4/tree/master/javascript/JavaScriptParser.g4
+grammar_file: https://github.com/antlr/grammars-v4/tree/master/javascript/
+JavaScriptParser.g4
 aliases:
```

Comment:
syntax error: could not find expected ':'

*Origin: YAMLLintBear, Section: `yaml`.*